packages/@aws-cdk/aws-cloudtrail/lib/cloudtrail.ts

```diff
@@ -79,6 +79,12 @@ export interface TrailProps {
    */
   readonly cloudWatchLogGroup?: logs.ILogGroup;
 
+  /**
+   * Role to use for CloudWatch logs delivery. Ignored if sendToCloudWatchLogs is set to false.
+   * @default - a new role is created and used.
+   */
+  readonly cloudWatchLogRole?: iam.IRole;
```

Comment:
should this be called `cloudWatchLogsRole`? in concert with `sendToCloudWatchLogs` and `cloudWatchLogsRetention`?

Reply:
Good point. `cloudWatchLogGroup` is a property of Cloudwatch Logs, while this role does not belong to Cloudwatch Logs but only provide permissions to it. What is about the name `sendToCloudWatchLogsRole`?

Reply:
According to the Cloudformation the property is named: `CloudWatchLogsRoleArn`, so cloudWatchLogsRole looks reasonable.